gallery/src/styles/NxList/NxListClickableLinksExample.tsx

```diff
@@ -39,8 +43,11 @@ const NxListClickableLinksExample = () =>
         <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
       </a>
     </li>
-    <li className="nx-list__item nx-list__item--clickable" aria-disabled="true">
-      <a href="#/pages/nx-alert" className="nx-list__link disabled">
+    <li className="nx-list__item nx-list__item--clickable">
+      <a href="#/pages/nx-alert"
+         onClick={evt => { evt.preventDefault(); }}
+         className="nx-list__link disabled"
+         aria-disabled="true">
```

Comment:
The main difference here is that the plugin pointed out that `aria-disabled` isn't valid on the list item itself. But it seems to be valid on the link.